concepticondata/conceptlists.tsv

```diff
@@ -39,12 +39,12 @@ Kessler-2001-200	Kessler, Brett	2001		200	basic	English	global		Kessler2001		Thi
 Gregersen-1976-217	Gregersen, Edgar A.	1976		217	basic	English	African		Gregersen1976	Gregersen1976	This concept list is based on a merger of several concept lists proposed by Swadesh. As the compiler mentions himself, the list is composed of all 200 items proposed in [Swadesh (1952)](:ref:Swadesh-1952-200), all additional items proposed in [Swadesh (1955)](:ref:Swadesh-1955-100), and [Swadeshs supplementary list of 100 items](:ref:Swadesh-1960-100). According to the annotation of the compiler, all items occuring in the 200-item list and the 100-item list are marked with one asterisk. All items occuring only in the 100-item list are marked with two asterisks, and all items makred with a preceding dash (—) occur only in the supplementary list.	110-116	
 Sato-2002-200	Sato, Tomomi	2002		200	areal,documentation	English	Ainu	http://hdl.handle.net/2115/34009	Sato2002	Sato2002	This list was taken from an article which cites basic vocabulary items from the Ainu dialect of Samani. It is not clear on which criteria the author selected the meaning list. He quotes work of Shirō Hattori ([1961](:bib:Hattori1961)), and lists data presented in an earlier dictionary compiled by Hattori, but it is not clear whether the selection of meanings was also based on previous work.	99-114	
 Lee-2013-200	Lee, S. and Hasegawa, T.	2013		200	basic	English,Japanese	Ainu		Lee2013		This list was used to study the evolution of the Ainu languages. It is almost completely identical with the list of [200 items by Hattori (1973)](:ref:Hattori-1973-200), but contains additional Japanese translations. The first author was so friendly to provide us with the concept list in digital form.		
-Swadesh-1954-97	Swadesh, Morris	1954		97	basic	English	American Indian languages		Swadesh1954	Swadesh1954	"According to Swadesh, this list of 97 items was extracted from a larger collection of 120 wordlists in American Indian languages. The 97 items which Swadesh labeled as &quot;non-cultural&quot; seem to be a compromise between coverage and useful items for language comparison, which would also explain the rather strange number of 97 concepts."	316-318	
+Swadesh-1954-97	Swadesh, Morris	1954		97	basic	English	American Indian languages		Swadesh1954	Swadesh1954	According to Swadesh, this list of 97 items was extracted from a larger collection of 120 wordlists in American Indian languages. The 97 items which Swadesh labeled as &quot;non-cultural&quot; seem to be a compromise between coverage and useful items for language comparison, which would also explain the rather strange number of 97 concepts.	316-318	
```

Comment:
This is LibreOffice (or something else) wreaking havoc with the quotation marks. Happens multiple times further down the list.

Reply:
ok, on it now.